src/parser/csql_grammar.y

```diff
@@ -24884,49 +24879,35 @@ dblink_column_definition_list
                PARSER_SAVE_ERR_CONTEXT ($$, @$.buffer_pos)
              DBG_PRINT}}
         ;
-    
+
 dblink_column_definition
-        : IdName primitive_type
+        : identifier primitive_type
         {{
-                PT_NODE *node = 0x00;
-                PT_NODE *p = parser_new_node (this_parser, PT_NAME);
-                if (p)
-                {
-                        int size_in;
-                        char *str_name = $1;
+                PT_NODE *node;              
 
-                        size_in = strlen(str_name);
+                node = parser_new_node (this_parser, PT_ATTR_DEF);
+                if (node)                            
+                {                    
+                        PT_NODE *dt;
+                        PT_TYPE_ENUM typ;
 
-                        PARSER_SAVE_ERR_CONTEXT (p, @$.buffer_pos)
-                        str_name = pt_check_identifier (this_parser, p,
-                                                        str_name, size_in);
-                        p->info.name.original = str_name;
-                                
-                        node = parser_new_node (this_parser, PT_ATTR_DEF);
-                        if (!node)
-                                parser_free_node(this_parser, p);
-                        else        
-                        {                    
-                                PT_NODE *dt;
-                                PT_TYPE_ENUM typ;
-
-                                node->type_enum = typ = TO_NUMBER (CONTAINER_AT_0 ($2));
-                                node->data_type = dt = CONTAINER_AT_1 ($2);
-                                node->info.attr_def.attr_name = p;
-
-                                if(typ == PT_TYPE_BLOB || typ == PT_TYPE_CLOB)
-                                        ; // TODO: error processing
-
-                                if (typ == PT_TYPE_CHAR && dt)
-                                        node->info.attr_def.size_constraint = dt->info.data_type.precision;
-                                if (typ == PT_TYPE_OBJECT && dt && dt->type_enum == PT_TYPE_VARCHAR)
-                                {
-                                        node->type_enum = dt->type_enum;
-                                        PT_NAME_INFO_SET_FLAG (node->info.attr_def.attr_name,
-                                                        PT_NAME_INFO_EXTERNAL);
-                                }
+                        node->type_enum = typ = TO_NUMBER (CONTAINER_AT_0 ($2));
+                        node->data_type = dt = CONTAINER_AT_1 ($2);
+                        node->info.attr_def.attr_name = $1;
+
```

Comment:
Wouldn't we need the code below?

```suggestion
			    node->info.attr_def.attr_type = PT_NORMAL
			    
```

Reply:
node = parser_new_node (this_parser, PT_ATTR_DEF); When executed, it is set as PT_NORMAL and comes out.